region/maridia/inner-yellow.json

```diff
@@ -872,7 +872,8 @@
                     "canUseFrozenEnemies",
                     {"or": [
                       "canWalljump",
-                      "h_canCrouchJumpDownGrab"
+                      "h_canCrouchJumpDownGrab",
```

Comment:
I still can't get the jump off the fish without `CrouchJumpDownGrab`

Reply:
Whoops. You're right. I was confusing myself